Description: Make dh_cargo happy about libcinnabar.a
Author: Mike Hommey <glandium@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: git-cinnabar/build.rs
===================================================================
--- git-cinnabar.orig/build.rs
+++ git-cinnabar/build.rs
@@ -269,4 +269,5 @@ fn main() {
     }
 
     println!("cargo:rerun-if-env-changed=CINNABAR_MAKE_FLAGS");
+    println!("dh-cargo:deb-built-using=cinnabar=0={}", env("CARGO_MANIFEST_DIR"));
 }
